main/sentry.py

```diff
@@ -29,14 +29,17 @@ def before_send(event, hint):
     return event
 
 
-def init_sentry(*, dsn, environment, version, log_level, heroku_app_name):
+def init_sentry(
+    *, dsn, environment, version, send_default_pii, log_level, heroku_app_name
+):
     """
     Initializes sentry
 
     Args:
         dsn (str): the sentry DSN key
         environment (str): the application environment
         version (str): the version of the application
+        send_default_pii (boo): enable sending PII data to associate users to errors
```

Comment:
Is this spooky or should this say "bool"?

Reply:
ha, typo